README.md

```diff
@@ -60,6 +66,57 @@ cp envs/env_vars_deactivate.sh <conda_prefix>/etc/conda/deactivate.d/env_vars.sh
 If necessary, create any missing directories manually.
 In case some lines in the environment scripts cause problems, you can edit the files to trouble-shoot.
 
+These operations are also automated in the `envs/set_vars.sh` script, which you can call via
+
+```console
+. envs/set_vars.sh <conda_prefix>
+```
+
+### Python environments
+
+There are multiple different environments for the python dependencies:
+
+| YAML file location           | Environment name    | Description                                                                                           |
+| ---------------------------- | ------------------- | ----------------------------------------------------------------------------------------------------- |
+| `envs/scib-pipeline.yml`     | `scib-pipeline`     | Base environment for calling the pipeline, running python integration methods and computing metrics   |
+| `envs/scIB-python-paper.yml` | `scIB-python-paper` | Environment used for the results in the [publication](doi: https://doi.org/10.1101/2020.05.22.111161) |
+
+The `scib-pipeline` environment is the one that the user activates before calling the pipeline.
+It needs to be specified under the `py_env` key in the config files under `configs/` so that the pipelien will use it for running python methods. Alternatively, you can specify `scIB-python-paper` as the `py_env`, so that the dependencies of the paper are used instead for integration runs. 
+Furthermore, `scib-pipeline` python environments require the R package [`kBET`](https://github.com/theislab/kBET) to be installed manually. This also requires that environment variables are set as described above, so that R packages are correctly installed and located. Once environment variables have been set, you can install `kBET`:
+
```

Comment:
is kBet installation not doable in your install script?

Reply:
or not because it requires also the cpp compilation?

Reply:
kBET is needed for the python environment, not the R environment. I don't currently have any python env installation script, but I might be able to add kBET as a pip dependency to the environment yaml file

Reply:
ah no, that's not possible, as it's an R package... Guess the user will have to stick with the manual install solution